kernel/aster-nix/src/fs/utils/page_cache.rs

```diff
@@ -224,10 +237,29 @@ enum PageState {
 
 /// This trait represents the backend for the page cache.
 pub trait PageCacheBackend: Sync + Send {
-    /// Reads a page from the backend.
-    fn read_page(&self, idx: usize, frame: &VmFrame) -> Result<()>;
-    /// Writes a page to the backend.
-    fn write_page(&self, idx: usize, frame: &VmFrame) -> Result<()>;
+    /// Reads a page from the backend asynchronously.
+    fn read_page_async(&self, idx: usize, frame: &VmFrame) -> Result<BioWaiter>;
```

Comment:
I think the `async` can be removed for async read and write, just use `read_page` and `write_page`.  The name `read_page_sync` and `write_page_sync` is enough

Reply:
FIXED